trigger_undp_kenya_s01_demog.py

```diff
@@ -0,0 +1,91 @@
+import argparse
+import json
+
+from core_data_modules.logging import Logger
+from dateutil.parser import isoparse
+from id_infrastructure.firestore_uuid_table import FirestoreUuidTable
+from rapid_pro_tools.rapid_pro_client import RapidProClient
+from storage.google_cloud import google_cloud_utils
+
+from src.lib import PipelineConfiguration
+
+log = Logger(__name__)
+
+rapid_pro_domain = "textit.in"
```

Comment:
I understand the need to land this soon, in a future pr/this pr lets get this from the pipeline config file so that we can easily port this to other Kenyan pipelines.

Reply:
Agreed that this is better coming from config. Unfortunately it wasn't trivial because I needed a way to specify which of the multiple RapidProSources to use, so will need to address this later

Reply:
Will add a TODO